src/stars.css

```diff
@@ -0,0 +1,8 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  padding-left: 2px;
+  padding-top: 2px;
+  padding-right: 6px;
```

Comment:
Redundant padding, you can remove it and update star size due to figma

Reply:
Figma file is different than reference file in test. You can't pass npm tests when you use Figma as reference